callisto/delivery/views.py

```diff
@@ -90,20 +92,31 @@ def submit_to_matching(request, report_id, form_template_name="submit_to_matchin
             if form.is_valid() and formset.is_valid():
                 try:
                     match_reports = []
+                    identifiers = []
                     for perp_form in formset:
-                        #enter into matching
+                        # enter into matching
                         match_report = MatchReport(report=report)
-                        match_report.contact_name = conditional_escape(form.cleaned_data.get('name'))
+                        match_report_text = {}
+
+                        perp_identifier = match_report_text['identifier'] = perp_form.cleaned_data.get('perp')
+                        match_report_text['perp_name'] = conditional_escape(perp_form.cleaned_data.get('perp_name'))
```

Comment:
Maybe a class that has a `__dict__` method? I like the class approach, since it provides something closer to type safety than what a bare dictionary gives you.